public/examples/ach.html

```diff
@@ -137,33 +138,24 @@
           event.preventDefault();
 
           try {
-            // disable the submit button as we await tokenization and make a payment request.
-            cardButton.disabled = true;
             achButton.disabled = true;
-            const token = await tokenize(paymentMethod, options);
-            const paymentResults = await createPayment(token);
-            displayPaymentResults('SUCCESS');
-
-            console.debug('Payment Success', paymentResults);
+            tokenize(paymentMethod, options);
           } catch (e) {
-            cardButton.disabled = false;
             achButton.disabled = false;
             displayPaymentResults('FAILURE');
             console.error(e.message);
           }
         }
 
-        const cardButton = document.getElementById('card-button');
-        cardButton.addEventListener('click', async function (event) {
-          await handlePaymentMethodSubmission(event, card);
-        });
-
-        const achButton = document.getElementById('ach-button');
-        achButton.addEventListener('click', async function (event) {
-          const paymentForm = document.getElementById('payment-form');
-          const achOptions = getACHOptions(paymentForm);
-          await handlePaymentMethodSubmission(event, ach, achOptions);
-        });
+        // Checkpoint 2.
```

Comment:
This checkpoint is directly at odds with the idea of this example being completely focused on ACH and ACH alone, I think checkpoint 1 was the card example.  We should pick a lane here - each payment method is independent, or the examples build on each other.

Reply:
good callout